OrbitLinuxTracing/TracerThread.h

```diff
@@ -16,6 +17,7 @@
 #include <regex>
 #include <vector>
 
+#include "../OrbitCore/TracepointCustom.h"
```

Comment:
just TracepointCustom.h

Reply:
Maybe I am missing something but this file is in OrbitLinuxTracing and TracepointCustom.h is in OrbitCore. By only letting "TracepointCustom.h" the file won't be recognised.

Reply:
You need to add OrbitCore to the dependencies of OrbitLinuxTracing in the CMakeLists.txt. The include dir in OrbitCore is then added to the appropriate search path during compilation.  

This is a way to enfore declaring dependencies in the build system.

Reply:
I looked closer into the code and I do not need anything from TracepointCustom after all. At least, I know what to do in the future to get rid of relative paths. Thank you so much for the explanations!